gnovm/tests/testdata/error_correct.txt

```diff
@@ -0,0 +1,18 @@
+# Test Error instruction correct
+
+exec $GNO test -verbose -root-dir=$ROOTDIR .
+
+stdout 'Machine\.RunMain\(\) panic: oups'
+stderr '=== RUN   file/x_filetest.gno'
+stderr '--- PASS: file/x_filetest.gno \(\d\.\d\ds\)'
+stderr 'ok      \./\. 	\d\.\d\ds'
+
+-- x_filetest.gno --
+package main
+
+func main() {
+	panic("oups")
+}
+
+// Error:
+// oups
```

Comment:
Each txtar file (txt is also an accepted extension) is executed in his own directory, so here the only file present for that test is `x_filetest.gno`.

Reply:
It looks great.

What do you think about moving this to `gnovm/cmd/gno/testdata`, since it’s testing the CLI?

Reply:
Good point. Ideally, I think those test should be added to `gnovm/cmd/gno/test_test.go`, and so this whole file should be rewritten using `testscripts`. Indeed I find the current format hard to read and maintain. The only good part is it fits in a single file, whereas with typescript there would be multiple small txtar files.

Reply:
> Good point. Ideally, I think those tests should be added to `gnovm/cmd/gno/test_test.go`, and so this whole file should be rewritten using `testscripts`. Indeed I find the current format hard to read and maintain. The only good part is it fits in a single file, whereas with typescript there would be multiple small txtar files.